src/js/tracker.js

```diff
@@ -1554,9 +1549,11 @@
 
 			// Send ping (to log that user has stayed on page)
 			var now = new Date();
+			var installingActivityTracking = false;
```

Comment:
Do we need `installingActivityTracking` as distinct from `activityTrackingInstalled`? 

We set both to false globally first, then at the same time we set them both to true if activity tracking is enabled, & we don't use them in conjunction - so I can't see where one would be true when the other is false.

Reply:
`installingActivityTracking` is defined in scope of `logPageView`, whereas `activityTrackingInstalled` is declared in for the `Tracker`. To maintain the old functionality of only starting page pings one the first page view, I need another boolean that only gets set to true on the first `logPageView()` (`installingActivityTracking` is only true when `activityTrackingInstalled` is false on first `logPageView()`, but it is false on subsequent called where `activityTrackingInstalled` is true). If I used `activityTrackingInstalled` then I would end up resetting the page ping timers on every `logPageView` which I only want to do if `resetActivityTrackingOnPageView` is true (new default).

I'll likely rip this out with the next MAJOR revision we make and reset page pings on every page view but as we want this to be a MINOR revision, I needed a way to maintain backwards compatibility in case anyone is relying on this (albeit broken imo) behaviour.

Reply:
Ah, I see where my confusion came from - I didn't follow the indentation properly. 

Yeah, this all makes a whole heap of sense, agreed on all fronts. Thanks for explaining!